internals/secrethub/signup.go

```diff
@@ -160,22 +162,28 @@ func (cmd *SignUpCommand) Run() error {
 
 	fmt.Fprintln(cmd.io.Stdout(), "Signup complete! You're now on SecretHub.")
 
-	fmt.Fprintln(cmd.io.Stdout(), "Setting up a start workspace...")
+	return createStartRepo(client, cmd.io.Stdout(), cmd.username, cmd.fullName)
+}
 
-	repoPath := secretpath.Join(cmd.username, "start")
-	_, err = client.Repos().Create(secretpath.Join(repoPath))
+// createStartRepo creates a start repository and write a fist secret to it, so that
+// the user can start by reading their first secret. This is intended to smoothen
+// onboarding.
+func createStartRepo(client secrethub.ClientInterface, w io.Writer, workspace string, name string) error {
+	fmt.Fprintln(w, "Setting up a start workspace...")
+	repoPath := secretpath.Join(workspace, "start")
+	_, err := client.Repos().Create(secretpath.Join(repoPath))
 	if err != nil {
 		return err
 	}
 
 	secretPath := secretpath.Join(repoPath, "hello")
-	message := fmt.Sprintf("Welcome %s! This is your first secret. To write a new version of this secret, run:\n\n    secrethub write %s", cmd.fullName, secretPath)
+	message := fmt.Sprintf("Welcome %s! This is your first secret. To write a new version of this secret, run:\n\n    secrethub write %s", name, secretPath)
```

Comment:
Spaces or tab(s) before the `secrethub write` text?

Reply:
I'd say spaces, as we use them in other places as well (e.g. root command help text).